xikolo-ios/Layouts/Base.lproj/TabCourses.storyboard

```diff
@@ -1333,6 +1319,6 @@
     </resources>
     <inferredMetricsTieBreakers>
         <segue reference="xR8-E6-ooy"/>
-        <segue reference="gMU-OF-QEH"/>
+        <segue reference="tbf-f6-wJw"/>
```

Comment:
@mathebox are you sure you wanted this? Be careful, I think this crashed automatic navigation bar handling once

Reply:
I think that's completely valid since I removed one segue and added another one.